chatbot/intents.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
could you add screenshot about how this is going to look like? also please link the issue into PR